test/categorical_algebra/FinSets.jl

```diff
@@ -29,7 +29,7 @@ h = FinFunction([3,1,2], 3)
 ########
 
 # Terminal object.
-@test ob(terminal(FinSet{Int})) == FinSet(1)
+@test ob(terminal(FinSet{Int,Int})) == FinSet(1)
```

Comment:
With the above changes to `terminal`, `product`, etc, we shouldn't have to change any of the unit tests.